File: src/kotlin/BelgianBeers/src/test/java/org/jetbrains/kotlincsharpdemo/D01_LoadJson.kt
package org.jetbrains.kotlincsharpdemo

import org.junit.Assert
import org.junit.Test

class D01_LoadJson {

  @Test
  fun load() {
    val sourceData = TestData.beerWwithNulls
    val repository = BeersRepository.fromFile(sourceData);

    Assert.assertTrue(repository.GetBeers().any())
  }

}

File: src/csharp/BelgianBeers/BelgianBeers/Tests/D01_LoadJson.cs
using System.Linq;
using System.Threading.Tasks;
using BelgianBeers.Repositories;
using BelgianBeers.Tests.Utilities;
using Xunit;

namespace BelgianBeers.Tests
{
    public class D01_LoadJson
    {
        [Fact]
        public void LoadsDataFromJsonFile()
        {
            var sourceData = TestData.DetermineDataPath("beerswithnulls.json");
            var repository = BeersRepository.FromFile(sourceData);
            
            Assert.True(repository.GetBeers().Any());
        }
    }
}